src/components/breadcrumb/breadcrumb.html

```diff
@@ -1,4 +1,4 @@
-<div class="bx--breadcrumb bx--breadcrumb--no-trailing-slash">
+<nav class="bx--breadcrumb bx--breadcrumb--no-trailing-slash" aria-label="breadcrumb">
```

Comment:
FYI: merged another breadcrumb PR and resolved a merge conflict here for you.
Question: Can you explain why this needs to be `nav` and why this needs an `aria-label`?